lib/theme.json

```diff
@@ -1,15 +1,5 @@
 {
 	"version": 2,
-	"behaviors": {
-		"blocks": {
-			"core/image": {
-				"lightbox": {
-					"enabled": false,
-					"animation": ""
-				}
-			}
-		}
-	},
```

Comment:
Can this just be removed or does it need a deprecation of some sort? Behaviors were [shipped](https://github.com/WordPress/gutenberg/releases/tag/v16.4.0) in GB 16.4 so never included in a WP release.

cc @ellatrix @gziolo @youknowriad

Reply:
@artemiomorales and I looked at whether this is used in production, and it looks like you can change that setting in the UI, which most likely gets reflected in what gets saved in the database. While it isn't mandatory for WordPress core, it would still be nice to provide a migration path for sites that have the latest Gutenberg plugin enabled. It shouldn't be that difficult to code something in the plugin to read `behaviors` object from the site and transfer the value to the new structure.

I guess the same applies to the block attribute that changes from `behaviors.lightbox` to `lightbox` in the Image block.

Reply:
In the followup PR: https://github.com/WordPress/gutenberg/pull/54071 we've added backwards-compatibility for users that might have been using the previous `behaviors.lightbox` syntax. 

In that PR (which is based on the current one) we're also now introducing a `lightbox.allowEditing` setting which allows themes to enable/disable the lightbox.